presto-main/src/main/java/com/facebook/presto/cost/CapDistinctValuesCountToTypeDomainRangeLength.java

```diff
@@ -46,12 +46,8 @@ public PlanNodeStatsEstimate normalize(PlanNode node, PlanNodeStatsEstimate esti
             estimate = estimate.mapSymbolColumnStatistics(
                     symbol,
                     symbolStatsEstimate -> symbolStatsEstimate.mapDistinctValuesCount(
-                            distinctValuesCount -> {
-                                if (!isNaN(distinctValuesCount)) {
-                                    return min(distinctValuesCount, domainLength);
-                                }
-                                return distinctValuesCount;
-                            }));
+                            distinctValuesCount -> min(distinctValuesCount, domainLength)
```

Comment:
I know this isn't what it was doing before, but should this be `minExcludeNaN()` instead?  Because if the distinctValuesCount is NaN, we presumably still have a cap.  Unless having "unknown" is more helpful to us than a max.